spring-batch-integration/src/main/java/org/springframework/batch/integration/async/AsyncItemWriter.java

```diff
@@ -23,14 +23,20 @@
 import java.util.List;
 import java.util.concurrent.Future;
 
-public class AsyncItemWriter<T> implements ItemWriter<Future<T>>, InitializingBean {
-	
+public class AsyncItemWriter<T> implements ItemStreamWriter<Future<T>>, InitializingBean {
+
 	private ItemWriter<T> delegate;
 
+	private boolean ignoreItemStream = false;
+
+	public void setIgnoreItemStream(boolean ignoreItemStream) {
+		this.ignoreItemStream = ignoreItemStream;
+	}
+

```

Comment:
We don't want to have this configurable.  If you don't want it delegated, don't implement `ItemStream`.